medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/DocumentController.cs

```diff
@@ -195,9 +197,9 @@ public async Task<IActionResult> UpdateClaimDmerOnDocument([FromRoute] string do
             var reply = _documentManagerClient.UpdateClaimDmer(request);
             if (reply.ResultStatus == Rsbc.Dmf.CaseManagement.Service.ResultStatus.Success)
             {
-                var caseDocument = _mapper.Map<CaseDocument>(reply.Item);
-
-                return Ok(caseDocument);
+                result = new DmerDocument();
+               
+                 return Ok(result);
```

Comment:
you are returning a new class here with no mapped properties from the reply

Reply:
Added automapper for DMER document